specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2021-10-01-preview/streamingjobs.json

```diff
@@ -571,6 +571,51 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.StreamAnalytics/streamingjobs/{jobName}/skus": {
```

Comment:
>skus [](http://example.com/codeflow?start=132&length=4)

this API needs to return a list of ARM objects and support individual gets on each object. 


i.e : 
GET .../streamingjobs/{jobname}/skus/{skuname}


Your other option if you only want to return a non-ARM formatted object is to change this into a POST method s.. i.e something like : 
POST .../streamingjobs/{jobname}/listskus

Reply:
I think the GET SKUs API is supposed to be list only, here is an example on VM get sku api:

https://docs.microsoft.com/en-us/rest/api/compute/resource-skus/list?tabs=HTTP

Reply:
Sorry about the confusion, this is correct. Approving the PR now.